styles/main/style.css

```diff
@@ -350,6 +352,55 @@ html {
   float: right;
 }
 
+/* Contacts section */
+.contacts {
+  background-color: var(--contacts-dark-gray);
+  margin-top: 140px;
+  text-align: center;
+  padding: 100px 0px;
+  position: relative;
+  z-index: 0;
+}
+.contacts__box{
+  position: relative;
+}
+.contacts__box:before{
+  content: "Hello";
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: var(--contacts-light-gray);
+  font-size: 30vw;
+  letter-spacing: 0.13em;
+  text-align: center;
+  z-index: -1;
+  width: 100vw;
+}
```

Comment:
There are two new problems. 

1. There is a horizontal overflow because .contacts__box:before goes outside of the parent
2. The text "Hello" doesn't align with the rest of the content.

![image](https://user-images.githubusercontent.com/68795596/209798927-cd3b196b-ca43-4a06-985f-4fd5895c7463.png)

Here is how you can fix it:

1. Set a max width to .contacts__box:before
2. Change the font-size(don't forget to do it on mobile too)